src/main/java/com/powsybl/openloadflow/equations/Equation.java

```diff
@@ -144,7 +161,10 @@ void initTarget(LfNetwork network, double[] targets) {
                 targets[row] = getBranchTarget(network.getBranch(num), PhaseControl.Unit.A);
                 break;
 
-            case ZERO_Q:
+            case LOAD_Q_DIFF:
+                targets[row] = getLoadQDiff(network, num, getData());
```

Comment:
If you add a target different than zero, it will be impossible to keep the reactive distibution given by the qPercent. This distribution is satisfied because this target is equal to zero in the master branch. But the idea I like in your PR is quite an optimization of the reactive power used in the network. I have noticed with your small test that we don't managed this and sometimes less reactive power is needed because the network could be help by loads of shunts. I am not sure that what you want to do here is local Indeed.